app/views/node.html

```diff
@@ -1,260 +1,6 @@
-            -
-          </p>
-        </div>
-
-        <div class="node-detail__subsection__cell">
-          <p class="cr-panel-block__item__header">
-            {{:: 'NODE.POST_RECOVERY' | translate }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="shardInfo.postrecovery > -1">
-            {{ shardInfo.postrecovery }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-hide="shardInfo.postrecovery > -1">
-            -
-          </p>
-        </div>
-
-      </div>
-    </div>
-
-    <div class="node-detail__subsection--medium">
-      <h2 class="">{{:: 'NODE.DISK_OPERATIONS' | translate }}</h2>
-      <div class="node-detail__subsection__detail cr-panel-block">
-        <div class="node-detail__subsection__cell">
-          <p class="cr-panel-block__item__header">
-            {{:: 'NODE.READ' | translate }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.iostats.rps > -1">
-            {{ node.iostats.rps|number:1 }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-hide="node.iostats.rps > -1">
-            -
-          </p>
-        </div>
-
-        <div class="node-detail__subsection__cell">
-          <p class="cr-panel-block__item__header">
-            {{:: 'NODE.WRITE' | translate }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.iostats.wps > -1">
-            {{ node.iostats.wps|number:1 }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-hide="node.iostats.wps > -1">
-            -
-          </p>
-        </div>
-
-        <div class="node-detail__subsection__cell">
-          <p class="cr-panel-block__item__header">
-            {{:: 'NODE.READS' | translate }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.iostats.rbps > -1">
-            {{ node.iostats.rbps|bytes }}/s
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-hide="node.iostats.rbps > -1">
-            -
-          </p>
-        </div>
-
-        <div class="node-detail__subsection__cell">
-          <p class="cr-panel-block__item__header">
-            {{:: 'NODE.WRITES' | translate }}
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.iostats.wbps > -1">
-            {{ node.iostats.wbps|bytes }}/s
-          </p>
-          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-hide="node.iostats.wbps > -1">
-            -
-          </p>
-        </div>
-
-      </div>
-    </div>
-
-  </div>
-
-</div>
+<div>
+	<div ng-class="$root.showNodeList ? '' : 'hide-node-list'">
+	  <node-list></node-list>
```

Comment:
I get the following error due to missing template: `Error: [$compile:tpload] Failed to load template: views/nodeList.html (HTTP status: 404 Not Found)`